### ISSUE TITLE
customHeaders are not setting

### ISSUE THREAD

**Issue by jpnarkinsky** [Vermonster/fhir-kit-client#138]
## Expected Behavior
When customHeaders are provided to the Client constructor as follows, they should appears in debug logs and be sent to the server.  

## Actual Behavior

The headers don't appear in the debug log and don't appear to be getting sent to the server.

## Steps to Reproduce the Problem

Here's sample code:

```
const clientId = '<redacted>';
const user = 'emp$REDACTED';
const pass = 'REDACTED';
const authorization= `Basic ${Buffer.from(`${user}:${pass}`).toString('base64')}`;

const Client = require('fhir-kit-client');

const fhirClient = new Client({
  baseUrl: 'https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/DSTU2',
  customHeaders: {
    "Epic-Client-ID": clientId,
    authorization,
  }
});

fhirClient.read({ resourceType: 'Patient', id: 'T81lum-5p6QvDR7l6hv7lf'}).then(console.log).catch(console.error)
```

And sample code:

```
DEBUG=fhir-kit-client:* node lib/test-headers.js 
  fhir-kit-client:info GET https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/DSTU2/Patient/T81lum-5p6QvDR7l6hv7lf +0ms
  fhir-kit-client:info Headers: [{}] +1ms
  fhir-kit-client:info Response: 401 +285ms
  fhir-kit-client:info [] +0ms
  fhir-kit-client:error Error: 401 +0ms
  fhir-kit-client:error     GET https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/DSTU2/Patient/T81lum-5p6QvDR7l6hv7lf +0ms
  fhir-kit-client:error     Headers: [] +0ms
  fhir-kit-client:error [""] +0ms
{
  response: { status: 401, data: '' },
  config: {
    method: 'get',
    url: 'https://fhir.epic.com/interconnect-fhir-oauth/api/FHIR/DSTU2/Patient/T81lum-5p6QvDR7l6hv7lf',
    headers: undefined
  }
}
```

I have tested with curl and it works.

## Specifications

  - Version: 1.6.2
  - Platform: Ubuntu 20.04 / node 14.15.5
  - Subsystem: I think this may be related to cross-fetch or node-fetch.  They are at 3.0.6 and 2.6.2 respectively.  I've tried different versions of them.
  
 This very well could be a problem on my side, but so far as I can tell it looks like a bug.  Thanks for any insight you can offer!

**Comment by jpnarkinsky** [comment]
It never fails that, once you do the work to file a bug report, you immediately figure out the issue. In this case, I was calling a different URL that what I had called with curl.   However, I will argue that there is a bug in that it doesn't log the headers properly.

Thanks!

**Comment by bkaney** [comment]
Okay, I can look at the logging to see how it can be improved.

**Comment by bkaney** [comment]
Fixed in 1.6.3. Give it a shot!
